src/vs/workbench/contrib/search/test/browser/searchViewlet.test.ts

```diff
@@ -72,8 +72,8 @@ suite('Search - Viewlet', () => {
 		const fileMatch = result.matches()[0];
 		const lineMatch = fileMatch.matches()[0];
 
-		assert.strictEqual(fileMatch.id(), `file:///${encodeURIComponent(getRootName())}/foo`);
-		assert.strictEqual(lineMatch.id(), `file:///${encodeURIComponent(getRootName())}/foo>[2,1 -> 2,2]b`);
+		assert.strictEqual(fileMatch.id(), URI.parse(`${getRootName()}/foo`).toString());
```

Comment:
`URI.parse` wants to take a full URI with a scheme, I think you just want `URI.file` here?

Reply:
oops, fixed!